package.json

```diff
@@ -39,7 +42,10 @@
     "clean:install": "npm run clean:all && npm install",
     "clean:obsolete-snapshots": "npm test -- -u",
     "compile": "lerna run compile",
+    "compile:heroku": "cd packages/terra-clinical-site && npm run compile:heroku",
     "deploy": "lerna run --scope terra-clinical-site deploy",
+    "heroku-prebuild": "npm install -g lerna@2.0.0-rc.4",
```

Comment:
You'll want to append `&& lerna init` in the `heroku-prebuild` script like we have in core: https://github.com/cerner/terra-core/blob/master/package.json#L47

This ensures if you update lerna, it is updated on the heroku instance correctly. We didn't run into this issue until we updated the version of lerna in terra-core.